frontend/src/components/validation/SearchForm.js

```diff
@@ -152,16 +158,60 @@ const SearchForm = (props) => {
     if (param === "order") {
       setDoRagnge(false);
     }
-    getFromOpenElisServer("/rest/user-test-sections", fetchTestSections);
+    switch(searchBy){
+      case "routine":
+        let testSectionId = new URLSearchParams(window.location.search).get(
+          "test_section_id"
```

Comment:
rename the  `testSectionId` in order to be consistent with the old ui

Reply:
Thanks, updated to `testSectionId`